lib/index.js

```diff
@@ -74,12 +78,6 @@ Appboy.prototype.loaded = function() {
  */
 
 Appboy.prototype.identify = function(identify) {
-  var userId = identify.userId();
-  // Appboy doesn't want you to call changeUser unless a userId is present because they do automatic anonymous user tracking on their end 
-  // https://www.appboy.com/documentation/Web/#setting-user-ids
-  if (!userId) return;
-  window.appboy.changeUser(userId);
```

Comment:
@anoonan i'd confirm w them before we merge this -- i feel like their point was that we shouldn't guard against the presence of `userId`. it seems like it should still call this if we have `userId`. How else would they know this?